src/js/facility-locator/containers/VAMap.jsx

```diff
@@ -109,6 +123,22 @@ class VAMap extends Component {
     }
   }
 
+  componentWillUnmount() {
+    this.listener();
```

Comment:
Yes I believe so; they use a const called `unlisten` in the docs https://github.com/ReactTraining/history/